src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="image-link" href="/">
+        <img
+        class="images"
+        src="images/logo.png"
+        alt="logo"
+        >
```

Comment:
```suggestion
        <img
          class="images"
          src="images/logo.png"
          alt="logo"
        >
```
you forget about indentation :)
Just FYI - no need to fix it here)